webpack/assets/javascripts/react_app/common/Tour/TourSelectors.js

```diff
@@ -0,0 +1,20 @@
+import { createSelector } from 'reselect';
+import { get } from 'lodash';
+
+export const selectTours = state => state.tours;
+
+export const selectActiveTour = createSelector(
+  selectTours,
+  tours => tours && Object.entries(tours).filter(tour => tour[1].running)[0]
+);
+
+export const selectIsAlreadySeen = (state, id) =>
+  get(state, `tours.${id}.alreadySeen`);
+
+export const selectIsRunning = (state, id) => get(state, `tours.${id}.running`);
+
+export const selectLoadingState = state => selectTours(state).loadingTour;
+const selectSessionStorage = (state, id) =>
+  sessionStorage.getItem(`TOUR_${id}`);
```

Comment:
I really like the idea of accessing `sessionStorage` with selectors, because it helps keep a single source of truth and we're not trying to sync anything in the Redux store.  That said, would it be better to stay consistent and write actions/thunks for the couple places elsewhere where you set the values?